src/main/java/seedu/address/ui/DetailView.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.ui;
+
+import seedu.address.model.event.Event;
+import seedu.address.model.person.Person;
+
+/**
+ * API of DetailView component
+ */
+public interface DetailView<T> {
```

Comment:
Is there supposed to be a full stop after each JavaDoc comment? 

As long as it's passing checkstyle it's fine, but let's KIV this for a future issue when we update all the docs